### PR TITLE
mark ipns as readonly

### PR DESCRIPTION
**Pull request by whyrusleeping**
Skip a few tests, and disable all paths for the fuse FS creating new content through that interface.

@jbenet filing this as a PR against yours, just as a formality.

### CODE REVIEW

File: fuse/ipns/ipns_test.go [hunk 1]
```diff
@@ -83,6 +83,7 @@ func setupIpnsTest(t *testing.T, node *core.IpfsNode) (*core.IpfsNode, *fstest.M
 
 // Test writing a file and reading it back
 func TestIpnsBasicIO(t *testing.T) {
+	t.Skip("Skipping until DAGWriter can be fixed.")
```
Comment (jbenet) [inline fuse/ipns/ipns_test.go]: `DAGWriter` or `DAGModifier`?  isn't `DAGWriter` what writes new files/objects?

**Comment by jbenet** [comment]
This changed hash/broken test is just my link block size change, correct?

```
expecting success: 
	HASH="QmbprabK1ucRoPLPns2zKtjAqZrTANDhZMgmcx6sDKPK92" &&
	echo "added $HASH mountdir/bigfile" >expected &&
	test_cmp expected actual
> diff -u expected actual
--- expected	2015-01-10 08:02:46.229535622 +0000
+++ actual	2015-01-10 08:02:46.199537341 +0000
@@ -1 +1 @@
-added QmbprabK1ucRoPLPns2zKtjAqZrTANDhZMgmcx6sDKPK92 mountdir/bigfile
+added QmSVxWkYfbJ3cowQUUgF4iF4CQd92vubxw7bs2aZAVRUD9 mountdir/bigfile
not ok 33 - ipfs add bigfile output looks good
#	
#		HASH="QmbprabK1ucRoPLPns2zKtjAqZrTANDhZMgmcx6sDKPK92" &&
#		echo "added $HASH mountdir/bigfile" >expected &&
#		test_cmp expected actual
#	
```

File: fuse/ipns/ipns_test.go (hunk 1 above)
Comment (whyrusleeping) [inline fuse/ipns/ipns_test.go]: whoops, youre right.

**Comment by whyrusleeping** [comment]
Yeah, that will just be your changes.

**Pull request merged**